packages/ckeditor5-ckbox/src/ckboximageedit/ckboximageeditcommand.ts

```diff
@@ -92,7 +99,10 @@ export default class CKBoxImageEditCommand extends Command {
 		return {
 			tokenUrl: ckboxConfig.tokenUrl,
 			onClose: () => this.fire<CKBoxImageEditorEvent<'close'>>( 'ckboxImageEditor:close' ),
-			onSave: () => this.fire<CKBoxImageEditorEvent<'save'>>( 'ckboxImageEditor:save' )
+			// Timeout for demo purpose.
+			onSave: ( data: CKBoxRawAssetDefinition ) => setTimeout( () =>
```

Comment:
I'm not sure if we want to delay this event. Let's discuss this. The possible options I see are:
1. Move the waiting (setTimeout for now) into `ckboxImageEditor:save` listener
2. Emit two events, one for save, one when the image has been processed.

We would have a ready point for placing the indicator.

Reply:
Yes, i think that two events is a good idea, but `ckboxImageEditor:save`  especially the `save` word is not fit for me. Better would be have two events like:

*   `ckboxImageEditor:beginSaveProcess`,
*   `ckboxImageEditor:finishSaveProcess`

WDYT?

BTW - don't know how far is @illia-stv with cksource/ckeditor5-commercial#5637